cstore_fdw.h

```diff
@@ -177,8 +186,10 @@ typedef struct StripeSkipList
  */
 typedef struct ColumnBlockData
 {
-	bool *existsArray;
-	Datum *valueArray;
+	StringInfo existBuffer;
+	StringInfo valueBuffer;
```

Comment:
One other argument for renaming ```ColumnBlockData``` to ```ColumnBlockDataBuffers``` is that we have used the member names ```existBuffer``` and ```valueBuffer``` inside it, so it is actually containing buffers.